webhooks/collector.go

```diff
@@ -109,3 +176,40 @@ func collectHooks() []WebhookQueue {
 
 	return destinations
 }
+
+func doAreasMatch(messageAreas, hookAreas []string) bool {
+	for _, area1 := range hookAreas {
+		hookArea, hookFence := splitIntoAreaAndFenceName(area1)
```

Comment:
This should have been split up on load of the config; this is being split every single webhook

Reply:
done